meta/init.go

```diff
@@ -2,23 +2,50 @@ package meta
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/XiaoMi/pegasus-go-client/idl/base"
 	"github.com/XiaoMi/pegasus-go-client/idl/replication"
 	"github.com/XiaoMi/pegasus-go-client/idl/rrdb"
 	"github.com/pegasus-kv/meta-proxy/rpc"
 )
 
-// Init meta API registration.
 func Init() {
+	initClusterManager()
+
 	rpc.Register("RPC_CM_QUERY_PARTITION_CONFIG_BY_INDEX", &rpc.MethodDefinition{
 		RequestCreator: func() rpc.RequestArgs {
 			return &rrdb.MetaQueryCfgArgs{
 				Query: replication.NewQueryCfgRequest(),
 			}
 		},
-		Handler: func(ctx context.Context, args rpc.RequestArgs) rpc.ResponseResult {
-			// TODO(wutao)
-			return nil
-		},
+		Handler: clusterManager.queryConfig,
 	})
 }
+
+func (m *ClusterManager) queryConfig(ctx context.Context, args rpc.RequestArgs) rpc.ResponseResult {
+	queryCfgArgs := args.(*rrdb.MetaQueryCfgArgs)
+	tableName := queryCfgArgs.Query.AppName
+	meta, err := m.getMetaConnector(tableName)
+	if err != nil {
+		return &rrdb.MetaQueryCfgResult{
+			Success: &replication.QueryCfgResponse{
+				Err: &base.ErrorCode{Errno: err.Error()},
```

Comment:
This is totally wrong. Errno should be defined within DsnErrorCode, rather than an arbitrary string.
`err` returned `getMetaConnector` could be any type of errors.

https://github.com/XiaoMi/pegasus-go-client/blob/master/idl/base/error_code.go

Reply:
Has changed，you can review again

(some other suggestion has not modify)